tests/layer_tests/pytorch_tests/test_topk.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+
+from pytorch_layer_test_class import PytorchLayerTest
+
+
+class TestTopK(PytorchLayerTest):
+    def _prepare_input(self):
+        return (self.input_tensor,)
+
+    def create_model(self, k, dim, largest, sort):
+        import torch
+
+        class aten_topk(torch.nn.Module):
+            def __init__(self, k, dim, largest, sort):
+                super(aten_topk, self).__init__()
+                self.k = k
+                self.dim = dim
+                self.largest = largest
+                self.sort = sort
+
+            def forward(self, input_tensor):
+                if self.dim is None:
+                    a, b = torch.topk(input_tensor, k=self.k, largest=self.largest, sorted=self.sort)
+                else:
+                    a, b = torch.topk(input_tensor, k=self.k, dim=self.dim, largest=self.largest, sorted=self.sort)
+                return b
+        ref_net = None
+
+        return aten_topk(k, dim, largest, sort), ref_net, "aten::topk"
+
+    @pytest.mark.parametrize(("input_tensor"), [
+        np.random.rand(7, 5, 5, 4),
+        np.random.rand(5, 5, 6, 7),
+        np.random.rand(5, 6, 6, 7, 8),
+        np.random.rand(7, 10, 5),
+        np.random.rand(5, 5, 6, 5),
+    ])
+
+    @pytest.mark.parametrize(("k"), [
+        3,
+        4,
+        1,
+        2,
+    ])
+
+    @pytest.mark.parametrize(("dim"), [
+        0,
+        1,
+        2,
+        -1,
+        None,
+    ])
+
+    @pytest.mark.parametrize(("largest"), [
+        True,
+        False,
+    ])
+    # For False it is hard to test because in Pytorch implementation
+    # there is not promise on the order of output values
```

Comment:
@rkazants I do not know if that is possible to support this test case since the unpredictability of the order of the output values is the expected thing. The values itself match but the order does not.